tools/sof-kernel-log-check.sh

```diff
@@ -164,8 +164,7 @@ ignore_str="$ignore_str"'|tpm tpm0: tpm_try_transmit: send\(\): error -5'
 ignore_str="$ignore_str"'|platform regulatory\.0: Direct firmware load for regulatory\.db failed with error -2'
 ignore_str="$ignore_str"'|cfg80211: failed to load regulatory\.db'
 ignore_str="$ignore_str"'|EXT4-fs \(nvme0n1p6\): re-mounted\. Opts: errors=remount-ro'
-ignore_str="$ignore_str"'|usb 2-3: Enable of device-initiated U1 failed\.'
-ignore_str="$ignore_str"'|usb 2-3: Enable of device-initiated U2 failed\.'
+ignore_str="$ignore_str"'|usb .-.+: (Enable|Disable) of device-initiated .+ failed\.'
```

Comment:
```suggestion
ignore_str="$ignore_str"'|usb .+: (Enable|Disable) of device-initiated .+ failed\.'
```

Reply:
we can ignore error messages for usb hub, but not all usb. for hub, I use `hub [23]-.+'`, and for usb, I update to use specific message `usb [23]-.+: Failed to suspend device, error` from #431.